structuralcodes/materials/constitutive_laws.py

```diff
@@ -302,6 +302,87 @@ def get_ultimate_strain(
         return (100, self._eps_u)
 
 
+class Sargin(ConstitutiveLaw):
+    """Class for Sargin constitutive law.
+    The stresses and strains are assumed negative in compression
```

Comment:
Please add a single line break before line 307.

Reply:
Done in [475c91c9](https://github.com/fib-international/structuralcodes/pull/88/commits/475c91c920213b94591c7b30431c873ad8bae3af) also for Parabola Rectangle and User Defined that had the same problem.

Reply:
Fixed in [4ed7ae4a](https://github.com/fib-international/structuralcodes/pull/88/commits/4ed7ae4ab4dbc86da6fdeb49223f5dbc93136a47)